src/layouts/PostLayout.js

```diff
@@ -18,6 +18,11 @@ import SEO from '../components/seo';
 library.add(fab, faCoffee);
 
 const PostLayout = ({ data }) => {
+  const styles = {
+    margin: 50,
```

Comment:
You can leave this how it was. I have already worked on it.

Reply:
ok great

Reply:
ok ,great